src/components/home/featuredprojects.tsx

```diff
@@ -0,0 +1,9 @@
+const FeaturedProjects = () => {
+  return (
+    <div className="flex h-screen w-screen items-center justify-center">
+      <p className="text-6xl font-bold text-gray-600">FEATURED PROJECTS</p>
+    </div>
```

Comment:
```suggestion
    <div className="flex justify-center">
      <p className="text-6xl font-bold text-gray-600">FEATURED PROJECTS</p>
    </div>
```
use acm colors within globals.css instead of default tailwind